apps/generic_terminal/generic_unix_shell.talon

```diff
@@ -0,0 +1,3 @@
+tag: user.generic_unix_shell
+-
+core {user.unix_utility}: user.terminal_run_unix_utility("{unix_utility}")
```

Comment:
Do you want this command to *run* the shell command, or just type it out? I'd prefer typing it out, because you might want to add options to the command afterward; you can always say "enter" afterwards to hit the enter key and run the command.

Either way, it seems like you don't need an action for this, you can just insert the command text.

Reply:
I meant text insertion of course, for pipeline composition and adding arguments, just as you said. The wording of the function was suboptimal.

> Either way, it seems like you don't need an action for this, you can just insert the command text.

D'oh, you are obviously right! Will simplify this.